tests/test_stdio.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+from pathlib import Path
+from polytracker import taint_dag, ProgramTrace
+
+
+@pytest.fixture
+def set_env_vars(monkeypatch):
+    monkeypatch.setenv("POLYTRACKER_STDOUT_SINK", "1")
+    monkeypatch.setenv("POLYTRACKER_STDERR_SINK", "1")
+
+
+@pytest.mark.program_trace("test_stdio.cpp")
+def test_tdfile(set_env_vars, program_trace: ProgramTrace):
+    assert isinstance(program_trace, taint_dag.TDProgramTrace)
+    stdout = Path("/dev/stdout")
+    stderr = Path("/dev/stderr")
+    headers = list(program_trace.tdfile.fd_headers)
+    outputs = list(map(lambda h: h[0], headers))
```

Comment:
I was confused for a moment here. There should only be two outputs, correct? After more careful reading, I noticed that outputs contain all fd-headers, not only outputs. Consider changing the name to  something like 'processed_filenames' (or something better you come to think of).